go.mod

```diff
@@ -5,7 +5,7 @@ go 1.23.0
 toolchain go1.24.1
 
 require (
-	github.com/ClickHouse/ch-go v0.66.1
+	github.com/ClickHouse/ch-go v0.66.2-0.20250716011122-97e525b8cb83
```

Comment:
updated to bring changes from https://github.com/ClickHouse/ch-go/pull/1074

this is using commit id, a new tag is not created.